### PR TITLE
Add a new blog 0305

### PR DESCRIPTION
**Pull request by booooodv**

### CODE REVIEW

File: number-of-TiDB-contributors-exceeded-400.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* 社区活动愈加丰富：
+	* 除了传统的 Infra Meetup、TiDB TechDay、TiDB DevCon 线下交流活动，颇具 Geek 精神的 TiDB Hackathon 已成功举办两届，诞生了很多奇思妙想的项目；
+	* TiDB 性能挑战赛第一季 落幕，第二季易用性挑战赛 正在进行中，接下来的 3 个月参赛选手将挑战“让 TiDB 变得更好用”；
+	* 本周末即将举办 Book Rush 活动，社区伙伴们挑战 48 小时合写一本 5～6 万字的 TiDB 技术书籍。
+
+* 社区学习版图不断拓宽：
+	* 面向开源社区开放的 Talent Plan 学习计划正在筹备 2.0 升级，整合更多优质学资源，让更多社区伙伴能学、善用，在开源世界、在分布式数据库领域更自在玩耍；
+	* PingCAP University 上也有面向 DBA 同学的学习资料库，包含线上视频课程还有线下培训，内容干货、形式丰富。
+
+* TiDB Map 正在完善：
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: https://github.com/pingcap/tidb-map

File: number-of-TiDB-contributors-exceeded-400.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	* 为了让每位 TiDB Developer 清晰了解 TiDB 社区的“玩法”，找到自己擅长的方向，我们整理了 Contribution Map，展示各个 Repo 中大家可以上手做的事情&参考资料，同时 Map 版图还在不断扩充中；
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: contribution map: https://github.com/pingcap/tidb-map/blob/master/maps/contribution-map.md

File: number-of-TiDB-contributors-exceeded-400.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+**这一切荣誉和成就都属于社区——**
+
+**TiDB Repo Contributor 突破 400,**  
+**Thanks for your contributions!**
+
+![](media/Number-of-TiDB-contributors-exceeded-400/1-400-contributor.png)
+
+**近一年 TiDB 开源社区架构和治理规则也在不断规范和升级迭代中：**
+
+* 社区自治组织和角色：
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: SIG  https://github.com/pingcap/community/tree/master/special-interest-groups

Comment (husiyu) [reply number-of-TiDB-contributors-exceeded-400.md]: WG : https://github.com/pingcap/community/tree/master/working-groups

File: number-of-TiDB-contributors-exceeded-400.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	* 目前为止，社区启动过 20 个 Working Group (WG) ， 成立了 5 个 Special Interest Group（SIG），它们都有了丰富的阶段性产出；
+	* 除了已有的 Maintainer、Committer、Contributor 角色之外， 新增了 Reviewer、Active Contributor 角色，让大家找到了“一步步升级打怪”的乐趣。
+
+* 社区活动愈加丰富：
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: Infra Meetup：https://pingcap.com/meetup/

Comment (husiyu) [reply number-of-TiDB-contributors-exceeded-400.md]: TiDB TechDay ： https://pingcap.com/community-cn/techday2019/

Comment (husiyu) [reply number-of-TiDB-contributors-exceeded-400.md]: TiDB DevCon： https://pingcap.com/community-cn/devcon2019/

File: number-of-TiDB-contributors-exceeded-400.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+	* 除了传统的 Infra Meetup、TiDB TechDay、TiDB DevCon 线下交流活动，颇具 Geek 精神的 TiDB Hackathon 已成功举办两届，诞生了很多奇思妙想的项目；
+	* TiDB 性能挑战赛第一季 落幕，第二季易用性挑战赛 正在进行中，接下来的 3 个月参赛选手将挑战“让 TiDB 变得更好用”；
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: TiDB 性能挑战赛第一季：https://pingcap.com/blog-cn/pcp-report-202002/

Comment (husiyu) [reply number-of-TiDB-contributors-exceeded-400.md]: 第二季易用性挑战赛：https://pingcap.com/blog-cn/TiDB-usability-challenge-program/

File: number-of-TiDB-contributors-exceeded-400.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+	* 本周末即将举办 Book Rush 活动，社区伙伴们挑战 48 小时合写一本 5～6 万字的 TiDB 技术书籍。
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: Book Rush：https://pingcap.com/blog-cn/blog-cn/tidb-book-rush-write-a-book-in-48-hours

File: number-of-TiDB-contributors-exceeded-400.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+* 社区学习版图不断拓宽：
+	* 面向开源社区开放的 Talent Plan 学习计划正在筹备 2.0 升级，整合更多优质学资源，让更多社区伙伴能学、善用，在开源世界、在分布式数据库领域更自在玩耍；
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: Talent Plan: https://university.pingcap.com/talent-plan/

File: number-of-TiDB-contributors-exceeded-400.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+	* PingCAP University 上也有面向 DBA 同学的学习资料库，包含线上视频课程还有线下培训，内容干货、形式丰富。
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: PingCAP University : https://university.pingcap.com/

File: number-of-TiDB-contributors-exceeded-400.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+* TiDB Map 正在完善：
+	* 为了让每位 TiDB Developer 清晰了解 TiDB 社区的“玩法”，找到自己擅长的方向，我们整理了 Contribution Map，展示各个 Repo 中大家可以上手做的事情&参考资料，同时 Map 版图还在不断扩充中；
+	* 针对 TiDB User Group，我们整理了 [TiDB 集群问题排查地图、性能调优地图](https://github.com/pingcap/tidb-map/) ，欢迎大家提出自己的建议。
```
Comment (husiyu) [inline number-of-TiDB-contributors-exceeded-400.md]: TiDB 集群问题排查地图：https://github.com/pingcap/tidb-map/blob/master/maps/diagnose-map.md

Comment (husiyu) [reply number-of-TiDB-contributors-exceeded-400.md]: 性能调优地图：https://github.com/pingcap/tidb-map/blob/master/maps/performance-map.png

File: number-of-TiDB-contributors-exceeded-400.md (hunk 1 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

File: number-of-TiDB-contributors-exceeded-400.md (hunk 2 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

File: number-of-TiDB-contributors-exceeded-400.md (hunk 3 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

File: number-of-TiDB-contributors-exceeded-400.md (hunk 4 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

File: number-of-TiDB-contributors-exceeded-400.md (hunk 5 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

File: number-of-TiDB-contributors-exceeded-400.md (hunk 6 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

File: number-of-TiDB-contributors-exceeded-400.md (hunk 7 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

File: number-of-TiDB-contributors-exceeded-400.md (hunk 8 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

File: number-of-TiDB-contributors-exceeded-400.md (hunk 9 above)
Comment (booooodv) [reply number-of-TiDB-contributors-exceeded-400.md]: done

**Comment by booooodv** [comment]
@YiniXu9506 PTAL

**Pull request merged**